great_expectations/expectations/core/expect_column_to_exist.py

```diff
@@ -30,33 +30,85 @@
 
 
 class ExpectColumnToExist(BatchExpectation):
-    """Expect the specified column to exist.
+    """Checks for the existence of a specified column within a table.
 
     expect_column_to_exist is a \
     [Batch Expectation](https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_batch_expectations).
 
+    BatchExpectations are one of the most common types of Expectation. They are evaluated for an entire Batch, and answer a semantic question about the Batch itself.
+
     Args:
         column (str): \
             The column name.
 
     Other Parameters:
-        column_index (int or None): \
+        column_index (int or None, optional): \
             If not None, checks the order of the columns. The expectation will fail if the \
```

Comment:
```suggestion
           When the value is not None, the column order is checked. The Expectation fails when the \
```
Update all similar instances for consistency.